extensions/script/lua/LuaScriptEngine.cpp

```diff
@@ -65,15 +65,32 @@ LuaScriptEngine::LuaScriptEngine()
 
 void LuaScriptEngine::eval(const std::string &script) {
   try {
-    lua_.script(script, sol::script_throw_on_error);
+    if (!module_directories_.empty()) {
+      auto appended_script = script;
+      for (const auto& module_dir : module_directories_) {
+        appended_script = "package.path = package.path .. \";" + module_dir + "/?.lua\"\n" + script;
+      }
+      lua_.script(appended_script, sol::script_throw_on_error);
```

Comment:
Please extract this logic and reuse it in `evalFile`

Reply:
Updated in c5bc09b98061f1d2295e38e4b1d63795c3eafdfd